src/v0/util/index.js

```diff
@@ -210,35 +214,6 @@ const getHashFromArrayWithValueAsObject = (
   return hashMap;
 };
 
-// NEED to decouple value finding and `required` checking
-// NEED TO DEPRECATE
-const setValues = (payload, message, mappingJson) => {
```

Comment:
How do we make sure the methods & classes being removed are not referred anywhere in cases of missing test coverage ?

Reply:
I'm making sure these are not being used by blanket search throughout the code.

Reply:
For now all the tests will fail in these individual PRs but we'll address all of that at once in the end.